### ISSUE TITLE
Aggiungere il Supporto per l'autori delle mod

### ISSUE THREAD

**Issue by arves100** [PlanetGeeks/mclauncher#4]
Ciao volevo sapere se nel LauncherV5 potresti aggiungere il supporto all'autore e il link delle varie mod.
Molte mod per i permessi delle ModPack richiedono un link che reindirizza al loro topic e il loro nome.
Grazie.

**Comment by PlanetGeeks** [comment]
Certo, era già prevista questa funzione, se hai altre idee non farti
problemi nel proporle
Il 25/dic/2014 13:44 "Arves100" notifications@github.com ha scritto:

**Comment by arves100** [comment]
Ok avrei un'altra idea.
Nel launcher V4 quando aggiungo uno splashscreen in .gif utilizzando il CustomJFrame mi viene una roba indescrivibile.
Nel V5 si potrebbe risolvere?

**Comment by PlanetGeeks** [comment]
Certo, per la grafica farò un bel sistema.
Il 25/dic/2014 15:04 "Arves100" notifications@github.com ha scritto:

**Comment by arves100** [comment]
Ok.

**Comment by arves100** [comment]
Ah si.. per bel sistema di grafica si intende anche le immagini a seconda della modpack che scegli con le animazioni?

**Comment by PlanetGeeks** [comment]
Diciamo che sará qualcosa di molto più complesso, ma semplice da gestire..
Avevo pensato la grafica come un sito, cioè
un file xml e relativo CSS dove puoi gestire i componenti mostrati a
schermo, la loro posizione, colori, proprietà e animazioni etc..
Il 25/dic/2014 16:02 "Arves100" notifications@github.com ha scritto:

**Comment by arves100** [comment]
sarebbe ottimo così mi imparo un po' di css e xml e posso fare un ottimo launcher. Grazie per le informazioni non vedo l'ora che arrivi
